### ISSUE TITLE
Joystick not working in Jog Mode (GRBL) v1.3.9

### ISSUE THREAD

**Issue by vazhure** [Sienci-Labs/gsender#432]
Joystick dont work in jog mode
Sender send only "\x85" command (Jog command appeared only once and moving continued after button release)
Only can change Jog mode: rapid, normal, precise using Joystick button
![gSender 1 3 9-EDGE 12_7_2023 13_08_43](https://github.com/Sienci-Labs/gsender/assets/124382776/c1530da4-4e40-4007-a93c-adf35082678a)

**Comment by vazhure** [comment]
seems it works when $20 = 1...
GrblController.js line 1732
                _let jogFeedrate;
                if ($20 === '1') {_

**Comment by walidkayhan** [comment]
Thanks for pointing this out, I've figured out the cause of the problem, it's this little bugger here:

https://github.com/Sienci-Labs/gsender/blob/c4a2e808177e14b09f373d28d9a5bd33dbf082c8/src/app/widgets/JogControl/index.jsx#L989-L992

That check is no longer needed so I removed it, gamepad button jogging should work moving forward but I will monitor its behavior before the next release :)
